src/render/locals.rs

```diff
@@ -169,9 +175,13 @@ fn pp_operand<'tcx>(
                     if let Ok(allocation) = ecx.memory.get_raw(ptr.alloc_id) {
                         let offset = ptr.offset.bytes();
                         if (offset as u128) < allocation.len() as u128 {
-                            let alloc_bytes = &allocation.inspect_with_undef_and_ptr_outside_interpreter(
-                                offset as usize .. (offset as usize).checked_add(len as usize).ok_or(err())?,
-                            );
+                            let alloc_bytes = &allocation
+                                .inspect_with_undef_and_ptr_outside_interpreter(
+                                    offset as usize
+                                        ..(offset as usize)
+                                            .checked_add(len as usize)
+                                            .ok_or(err())?,
+                                );
```

Comment:
This has become worse in my opinion.

Reply:
Any tips on how to prevent it (other than `#[rustfmt::skip]`, where I prefer suboptimal formatting over skip attributes)?

Reply:
The argument could be extracted to a new variable. (`alloc_range`?)